src/OTPublisher.js

```diff
@@ -1,7 +1,9 @@
+import 'react-native-get-random-values';
```

Comment:
Thanks for the PR, @xseignard. I don't see the `react-native-get-random-values` module being used. Can we remove it from the PR (here and in other files)?

Reply:
Wait ... I am now re-reading your comment about the crypto.getRandomValues polyfill. Let me test.

Reply:
Hello, if you want to get rid of `react-native-get-random-values` then you'll have to replace `uuid` package by some implementation that doesn't rely on `crypto.randomUUID`.

`crypto.randomUUID` ain't available in JSC or Hermes but used in recent versions of `uuid`, hence the need of the `react-native-get-random-values`

If you can live with insecure random generation, then I'd advise you to drop `uuid` and build some custom `uuidv4` on top of https://github.com/LinusU/react-native-get-random-values/blob/master/index.js#L8

But I really don't think it's worth the hassle only to support `expo@39-`. It is deprecated for a looong time already and people still using it would just keep their current `react-native-opentok` version.

Hope this helps

Reply:
@xseignard when I test this, I get an error (running in Android emulator or iOS Simulator):

```
Native module not found
global.ExpoModules.ExpoRandom.getRandomBase64String(byteLength)
```

![image](https://github.com/user-attachments/assets/9a8a80eb-d54a-443f-8988-7b4846eaf6ab)

Note that I am using expo 51.0.34:

```
npm show expo version
51.0.34
```

Reply:
I'm investigating other libraries. Maybe we can use `react-native-uuid`.
.

Reply:
Hey 👋 
Thanks for asking!
It seems it would be a good alternative and even better than my solution as it would not break expo versions below 39.

Thanks for digging! 🙏

Reply:
Great. Please feel free to review this PR @xseignard : https://github.com/opentok/opentok-react-native/pull/781. It includes your commits, so you will be credited.